docs/src/modules/brandingTheme.ts

```diff
@@ -305,7 +305,7 @@ export const getDesignTokens = (mode: 'light' | 'dark') =>
         fontWeight: 700,
       },
       allVariants: {
-        scrollMarginTop: 'calc(var(--MuiDocs-header-height) + 72px)',
+        scrollMarginTop: 'calc(var(--MuiDocs-header-height) + 32px)',
```

Comment:
32px is this distance:

<img width="434" alt="Frame 1 (1)" src="https://user-images.githubusercontent.com/3165635/233842161-4a9f1af2-3c23-4773-aab6-e11244f1f8bb.png">

⚠️ not to change